internal/status/checksum.go

```diff
@@ -32,6 +33,16 @@ func (c *Checksum) IsUpToDate() (bool, error) {
 		return false, err
 	}
 
+	generates, err := glob(c.Dir, c.Generates)
```

Comment:
Hi @topecongiro,

I just reminded that have no globs listed at `generates:` with `method: checksum` is a totally valid use case.

For this reason, we should only do the checks below if the user has informed any glob on `generates:`:

```go
if len(c.Generates) > 0 {
	// do all the stuff
}
```

Reply:
Done!